docs/published/main.tf

```diff
@@ -11,6 +15,10 @@ module "dcos" {
   source  = "dcos-terraform/dcos/aws"
   version = "~> 0.1"
 
+  providers = {
+    aws = "aws"
+  }
+
```

Comment:
ok perfect the example file has this already included. It should also be included in the code example above

Reply:
@bernadinm thanks for the example. This request for change was to update .md files to contain this example.

Reply:
ah! thank you!

Reply:
@bernadinm still missing in published/README.md